katalog/fluentd/prometheus.conf

```diff
@@ -1,11 +0,0 @@
-# Prometheus metric exposed on 0.0.0.0:24231/metrics
```

Comment:
Why are we dropping Prometheus metrics from fluentd?

Reply:
This setting is on the main fluentd.conf file

Reply:
Sorry, I totally missed it.